# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../kbarcode/compile-fix.patch
# Copyright (C) 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur kbarcode-1.9.8-orig/kbarcode/dstextedit.h kbarcode-1.9.8/kbarcode/dstextedit.h
--- kbarcode-1.9.8-orig/kbarcode/dstextedit.h	2005-06-17 17:41:17.000000000 +0000
+++ kbarcode-1.9.8/kbarcode/dstextedit.h	2005-12-21 14:01:46.000000000 +0000
@@ -14,6 +14,8 @@
  *   (at your option) any later version.                                   *
  *                                                                         *
  ***************************************************************************/
+#ifndef _DS_TEXT_EDIT_H_
+#define _DS_TEXT_EDIT_H_
 
 #include <qnamespace.h> 
 
@@ -47,4 +49,5 @@
      */
     //bool cursorIsInToken();
 };
+#endif
 
